website/docs/advanced-concepts/granular-access-control/roles.md

```diff
@@ -49,7 +49,7 @@ This section contains permissions related to the actions a user can perform on d
 | | <div style= {{width:"180px"}}> **Execute** </div> | <div style= {{width:"180px"}}> **Create** </div>| <div style= {{width:"180px"}}> **Edit** </div> | <div style= {{width:"180px"}}> **Delete** </div> | <div style= {{width:"180px"}}> **View** </div> |
 | --- | --- | --- | --- | --- | --- |
 | **Workspace** | Execute queries on any datasource in the workspace. | Create queries on any datasource in the workspace. | Edit any datasource in the workspace. | Delete any datasource in the workspace. | View any datasource in the workspace. |
-| **Environments** | Execute queries for staging and production environment.  | - | - | - | - |
+| **Environments** | Execute queries for different environments.  | Create queries for different environments.  | Edit queries for different environments. |  Delete queries for different environments.| View any environments in the workspace. |
```

Comment:
CRUD permissions defined for environments are related to the environments itself and not queries. We don't support queries within environments at all. Please update this accordingly.